utils/install_jq.sh

```diff
@@ -0,0 +1,7 @@
+
+if [ `uname -s` == 'Linux' ]; then
+  # this only works for Debian & Ubuntu for now
+  sudo apt-get install jq
+elif [ `uname -s` == 'Darwin' ]; then
+  brew install jq
+fi
```

Comment:
```suggestion
if [ `lsb_release -d | awk '{print $2}'` == 'Ubuntu' ]; then
  sudo apt install jq
elif [ `uname -s` == 'Darwin' ]; then
  brew install jq
else
  echo 'Unknown platform.'; exit -1
fi
```

Reply:
https://github.com/chef-boneyard/os_release I think we can try using this to detect linux distribution